### PR TITLE
GlobalMarineNet.XWeb version 1.7

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107506)

### CODE REVIEW

File: manifests/g/GlobalMarineNet/XWeb/1.7/GlobalMarineNet.XWeb.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GlobalMarineNet.XWeb
+PackageVersion: 1.7
+PackageLocale: en-US
+Publisher: GlobalMarineNer
```
Comment (Trenly) [inline manifests/g/GlobalMarineNet/XWeb/1.7/GlobalMarineNet.XWeb.locale.en-US.yaml:7]:
```suggestion
Publisher: GlobalMarineNet
```

**Review by Trenly — CHANGES_REQUESTED** [review]
This package doesn't write the Publisher or DisplayVersion to the apps and features entries and so users might not be able to upgrade it

**Comment by stephengillie** [comment]
This package appears to install Firefox.

**Comment by upintheairsheep** [comment]
> This package appears to install Firefox.

Screenshot please.

**Comment by stephengillie** [comment]
> > This package appears to install Firefox.
> 
> Screenshot please.

image

**Comment by upintheairsheep** [comment]
> > > This package appears to install Firefox.
> > 
> > 
> > Screenshot please.
> 
> image

OK, yes it's just firefox I guess.

**Comment by upintheairsheep** [comment]
So, close this? or merge?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> So, close this? or merge?

It's probably best to close it since we have Firefox in the package repository already.

**Comment by upintheairsheep** [comment]
It’s just Firefox.

**Pull request closed without merging**